CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Tests
+- CompanyServiceImpl - validate the proper mapping behaviour of method "readDetails"
+
```

Comment:
There is a `Tests` section at line 34, you can move this change there

Reply:
oops..ill fix that.